overcooked_ai_py/mdp/overcooked_env.py

```diff
@@ -211,6 +220,7 @@ def get_rollouts(self, agent_pair, num_games, display=False, final_state=False,
             trajectories["ep_dones"].append(dones)
             trajectories["ep_infos"].append(infos)
             trajectories["ep_returns"].append(tot_rews_sparse)
+            trajectories["ep_returns_shaped"].append(tot_rews_sparse + tot_rews_shaped * reward_shaping)
```

Comment:
Also here

Reply:
I like your solution to this ("define a metadata_fn that can extract the shaped_reward information...") -- very neat :)